services/vision/mlvision/detector.go

```diff
@@ -92,8 +93,7 @@ func attemptToBuildDetector(mlm mlmodel.Service) (objectdetection.Detector, erro
 				utils.Clamp(locations[4*i+getIndex(boxOrder, 2)], 0, 1)*float64(origW),
 				utils.Clamp(locations[4*i+getIndex(boxOrder, 3)], 0, 1)*float64(origH)
 			rect := image.Rect(int(xmin), int(ymin), int(xmax), int(ymax))
-			labelNum := int(categories[i])
-
+			labelNum := int(utils.Clamp(categories[i], 0, math.Inf(1)))
```

Comment:
do you actually want to clamp to infinity? You can also specify the biggest Int possible instead `math.MaxInt`

Reply:
Changed to MaxInt ty